.gitignore

```
@@ -24,3 +24,14 @@ id_rsa
 *.tgz
 *.tar
 *.gz
+
+# Tmpl
+tests/.coverage
+tests/coverage.xml
+.coverage
+coverage.xml
+dist/
+blueprint.egg-info/
+build/
+blueprint/__pycache__/
+tests/__pycache__/
```

```suggestion
__pycache__/
```

see [github template](https://github.com/github/gitignore/blob/218a941be92679ce67d0484547e3e142b2f5f6f0/Python.gitignore#L2)